src/server.cpp

```diff
@@ -3182,17 +3182,28 @@ std::string Server::getStatusString()
 	os << " | clients: ";
 	if (m_env) {
 		std::vector<session_t> clients = m_clients.getClientIDs();
+
+		// Create vector to store player names
+		std::vector<std::string> playerNames;
```

Comment:
`player_names` btw

Reply:
"Why?", you might ask. https://dev.minetest.net/Code_style_guidelines#Miscellaneous -> "snake_case for local variables".

Reply:
Ahh, got it